en/simulation/ignition_gazebo.md

```diff
@@ -18,44 +18,48 @@ See [Simulation](../simulation/README.md) for general information about simulato
 ## Installation (Ubuntu Linux)
 
 :::note
-These instructions were tested on Ubuntu 18.04 and Ubuntu 20.04
+These instructions were tested on Ubuntu 18.04, Ubuntu 20.04, and Ubuntu 22.04
 :::
 
 1. Install the usual [Development Environment on Ubuntu LTS / Debian Linux](../dev_setup/dev_env_linux_ubuntu.md).
-1. Install Ignition Gazebo following the [installation instructions](https://github.com/Auterion/px4-simulation-ignition#readme) (`sudo` may be required):
```

Comment:
FYI @Jaeyoung-Lim FYI I removed this link as the Auterion docs did not match the updates.

More generally, where possible all docs should go in the guide. Sometimes they might go in PX4 tree. But linking to docs in Auterion tree is annoying.

But these